src/plugins/vis_type_timeseries/public/application/components/vis_types/_vis_types.scss

```diff
@@ -2,7 +2,9 @@
   display: flex;
   flex-direction: column;
   flex: 1 1 100%;
-  padding: $euiSizeS;
+  border-width: $euiSizeS;
+  border-style: solid;
+  border-color: transparent;
```

Comment:
I look at this and think "Why not padding?" Can you please add a comment here as to why it's necessary to do it this way so no one tries to optimize by converting it back to padding?

Reply:
Yeah this is a workaround for [this](https://github.com/elastic/kibana/pull/69695#issuecomment-648708584) issue.

![image](https://user-images.githubusercontent.com/19007109/85729630-98186b00-b6be-11ea-94d6-65fe43db2fdc.png)

Bascially, the containing element for TSVB was applying a background color and padding. This was fine when elastic charts background was transparent but not anymore.

I think this can be cleaned up when we allow padding around the whole chart in the `Theme` but for now changing the `theme.chartMargins` does not include the legend.

Reply:
Yeah no worries. Just suggesting adding a comment in the SASS file to keep others aware if they jump in this file later.

Reply:
730c2e9